client/my-sites/site-settings/site-owner-transfer/site-owner-transfer.tsx

```diff
@@ -0,0 +1,41 @@
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import ActionPanel from 'calypso/components/action-panel';
+import FormattedHeader from 'calypso/components/formatted-header';
+import HeaderCake from 'calypso/components/header-cake';
+import Main from 'calypso/components/main';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import StartSiteOwnerTransfer from './start-site-owner-transfer';
+
+const SiteOwnerTransfer = () => {
+	const selectedSiteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const selectedSiteSlug = useSelector( ( state ) => getSelectedSiteSlug( state ) );
+
+	const translate = useTranslate();
+	if ( ! selectedSiteId || ! selectedSiteSlug ) {
+		return null;
+	}
+
+	return (
+		<Main>
+			<FormattedHeader
+				headerText={ translate( 'Site Transfer' ) }
+				subHeaderText={ translate( 'Transfer your site to another WordPress.com user.' ) }
+				align="left"
+			/>
+			<PageViewTracker
+				path="/settings/start-site Transfer/:site"
```

Comment:
I know this path was introduced in a different PR, but I just noticed this issue:
```suggestion
				path="/settings/start-site-transfer/:site"
```